docs/ref/csrf.txt

```diff
@@ -132,6 +132,12 @@ The above code could be simplified by using the `JavaScript Cookie library
     the DOM. Regardless, you're guaranteed to have the cookie if the token is
     present in the DOM, so you should use the cookie!
 
+.. note::
+
+    If you activate :setting:`CSRF_USE_SESSIONS`, this method does not work and
+    need to make sure you always include the CSRF token somewhere in your HTML
+    read the token from the DOM with JavaScript.
+
```

Comment:
The possible downside I see right now is that it would only work as expected if ``CSRF_COOKIE_*`` settings are set to the same constraints (domain, path, …) as ``SESSION_COOKIE_*``. So we'd have to ignore completely the ``CSRF_COOKIE_*`` settings in this case, which could lead to confusion.